root/edit/details/AddWork.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+
+ type AddWorkEditProps = {
```

Comment:
To avoid confusion, I think I'd only include "props" in the name if it's for something passed directly to a react component (not just any type that appears within props).

```suggestion
 type AddWorkEdit = {
```